### ISSUE TITLE
Beamline Macros

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/nabs#34]
## Expected Behavior

There is some proliferation of beamline macros (not plans) in the various hutch-python files. Do these belong in NABS?

## Current Behavior

Macros are scattered among the various hutch python installs

## Context

https://github.com/pcdshub/Bug-Reports-and-Requests/issues/31

**Comment by ZryletTC** [comment]
Still kinda unclear what kind of macros this refers to. @ZLLentz  could you give an example please?
I'm guessing this is different than the macros.py files that are in cxi, xcs, and mfx as those just look like extended beamline.py files. MFX's seems to be the only one with actual 'macros': `laser_in()` and `laser_out()` although those are mfx-specific (actually, CXI has some similar things too but it's a mess).

**Comment by ZLLentz** [comment]
No, this is exactly the macros.py items if there are any in common. It's not like I bothered to read those files, so if there is truly no commonalities then we can close this issue.
